src/tasks/netcore/updateBlazorManifest.ts

```diff
@@ -69,6 +69,7 @@ async function transformBlazorManifest(context: DockerRunTaskContext, inputManif
     if (!inputManifest || // Input manifest can't be empty/undefined
         !outputManifest || // Output manifest can't be empty/undefined
         !(await fse.pathExists(inputManifest)) || // Input manifest must exist
+        !(await fse.stat(inputManifest)).isFile() || // Input manifest must be a file
```

Comment:
We may not need the preceding `pathExists` call; I need to see what `stat` returns if the file does not exist.

Reply:
Nope, `stat` fails if the file doesn't exist. Need to keep `pathExists`.